browser-test/src/applicant/northstar_applicant_question_translation.test.ts

```diff
@@ -202,4 +203,15 @@ test.describe('Admin can manage translations', {tag: ['@northstar']}, () => {
 
     expect(await page.innerText('main form')).toContain('miembro de la familia')
   })
+
+  test.describe('Language Selector Visibility', () => {
+    test('shows language selector when multiple languages are enabled', async ({
+      page,
+    }: {
+      page: Page
+    }) => {
+      const selector = page.locator('[data-testid="languageSelector"]')
```

Comment:
(blocking) Please use the `page.getByTestId` method instead of a raw `page.locator`. See [getByTestId docs here](https://playwright.dev/docs/api/class-page#page-get-by-test-id).

Ideally instead of adding a `data-testid` though if it can be selected with `page.getByRole` that would be better, but `getByTestId` is fine if there's not a better alternative. See [recommended locators docs](https://playwright.dev/docs/locators#quick-guide).